### PR TITLE
made all tests completed

### PR DESCRIPTION
**Pull request by oleksii-manzik**

### CODE REVIEW

File: python_functionality/homework.py [hunk 1]
```diff
@@ -14,7 +14,12 @@ def task_1_fix_names_start_letter(data: DT) -> DT:
         fix_names_start_letters([{'name': 'Alex', 'age': 26}, {'name': 'denys', 'age': 89}])
         >>> [{'name': 'Alex', 'age': 26}, {'name': 'Denys', 'age': 89}]
     """
-    pass
+    return [
+        {
+            'name': d['name'].capitalize(),
+            'age': d['age']
+        } if d.get('name') else {'age': d['age']} for d in data
+    ]
```
Comment (dimigor) [inline python_functionality/homework.py]: do not format this so

File: python_functionality/homework.py [hunk 2]
```diff
@@ -25,7 +30,9 @@ def task_2_remove_dict_fields(data: DT, redundant_keys: List[str]) -> DT:
        remove_dict_field([{'name': 'Alex', 'age': 26}, {'name': 'denys', 'age': 89}], 'age')
         >>> [{'name': 'Alex'}, {'name': 'denys'}]
     """
-    pass
+    return [
+        {k: v for k, v in d.items() if k not in redundant_keys} for d in data
+    ]
```
Comment (dimigor) [inline python_functionality/homework.py]: do not move `[ ]` so

File: python_functionality/homework.py (hunk 1 above)
Comment (oleksii-manzik) [reply python_functionality/homework.py]: how am i supposed to format it if length of line is more than 80 characters?

Comment (oleksii-manzik) [reply python_functionality/homework.py]: fixed

File: python_functionality/homework.py (hunk 2 above)
Comment (oleksii-manzik) [reply python_functionality/homework.py]: fixed

**Review by dimigor — APPROVED** [review]

**Pull request merged**